src/index.js

```diff
@@ -8,10 +8,8 @@ import './index.css';
 
 createRoot(document.getElementById('root')).render(
   <Provider store={store}>
-    <React.StrictMode>
-      <BrowserRouter>
-        <Layout />
-      </BrowserRouter>
-    </React.StrictMode>
+    <BrowserRouter>
```

Comment:
What was the reason behind this config removal?

Reply:
Becouse  we had the problem that useEffect run two times in dispatch() ...
[  https://flaviocopes.com/react-useeffect-two-times/](url)

Reply:
Another Solutions ?

Reply:
i'll need to sit and check the code for a while to find a two times render component, it's prefferable to re-render some component than avoid checking on a lot of this we are not doing manually, like fragments and components lifecycles